filebeat/harvester/processor/multiline.go

```diff
@@ -213,20 +215,23 @@ func (mlr *MultiLine) startNewLine(l Line) Line {
 	retLine := mlr.pushLine()
 	mlr.addLine(l)
 	mlr.ts = l.Ts
+	mlr.fields = l.Fields
 	return retLine
 }
 
 func (mlr *MultiLine) pushLine() Line {
 	content := mlr.content
 	sz := mlr.readBytes
+	fields := mlr.fields
```

Comment:
For simplicity I was thinking that all fields besides the `message_key` are taken from the first event. This should be good enough for uses cases similar to the docker one. I should probably put this somewhere in the docs somewhere.